examples/sample-app/README.md

```diff
@@ -118,6 +137,8 @@ the present working directory is the same directory as this README.
       you'll see the page update as you deploy objects into the project
       and run builds.
 
+         **VAGRANT USERS**:  The web console is not available outside of the Vagrant box; move on to the next step.
```

Comment:
Actually it is, all you have to do is to run `openshift start --public-master=<your_vagrant_ip>`. I was having the same problem but asked @jwforres for that (see [here](https://botbot.me/freenode/openshift-dev/2015-02-26/?msg=32905928&page=3)). Then locally you access the web UI with `http://<your_vagrant_ip>:8444`